packages/react-components/src/organisms/SharedResearchOutputHeaderCard.tsx

```diff
@@ -83,7 +83,19 @@ const SharedResearchOutputHeaderCard: React.FC<SharedResearchOutputHeaderCardPro
             </li>
             {subTypes.map((subType, i) => (
               <li css={typeStyles} key={`subtype-${i}`}>
-                <TagLabel>{subType}</TagLabel>
+                <TagLabel>
+                  <span
+                    css={{
+                      display: '-webkit-box',
```

Comment:
Yeah I don't think this is right. It should be something with `white-space: nowrap`, like UsersList and TeamsList which also abbreviate with an ellipsis once too long for the line. It should also be part of `TagLabel` itself right, because the current behavior of `TagLabel` when the content is just broken and not desirable in any case afaik